charts/sonarqube/templates/change-admin-password-hook.yml

```diff
@@ -0,0 +1,28 @@
+apiVersion: batch/v1
```

Comment:
Might want to wrap the file in an if block in order to turn the functionality on/off

Reply:
done